src/popup/components/NameItem.vue

```diff
@@ -185,7 +185,10 @@ export default defineComponent({
     const account = useGetter<IAccount>('account');
 
     const nameEntry = computed<IName | null>(() => root.$store.getters['names/get'](props.name));
-    const hasPointer = computed(() => nameEntry.value?.pointers?.accountPubkey);
+    const hasPointer = computed<boolean>(() => !!nameEntry.value?.pointers?.accountPubkey);
```

Comment:
When typing computed properties it is better to type the return value of the callback as it will better deal with checking the logic. Now it just informs the parser what would be the returned value.

So instead of
```
computed<Something>(() => ...
```
it's better to
```
```
computed((): Something => ...
```